docs/customElementCss.md

```diff
@@ -0,0 +1,174 @@
+
+The true benefit with encapsulation of CSS and custom elements is that you do not need to be concerned about global styles having side effects on your element when used in any context of any website. While naming conventions and using parent selectors have become a staple of web development over the years, these techniques are not necessary for UI stability in custom elements. But they do come in handy for maintaining the sanity of your CSS. To that end, Auro has a series of [CSS development conventions](https://auro.alaskaair.com/webcorestylesheets/conventions) that we highly recommend.
+
+
+## :host selector
+
+With encapsulation comes the `:host` CSS pseudo-class. This special selector in the CSS world is... _" the most powerful selector in the universe... "_ You could draw a parallel between `:host` and `{this}` in JavaScript.
+
+> The `:host` CSS pseudo-class selects the shadow host of the shadow DOM containing the CSS it is used inside — in other words, this allows you to select a custom element from inside its shadow DOM.
+
+In practical terms, what does that mean? In other front-end frameworks, it is commonplace to wrap the scope of the component's HTML in a `div` and add a class. You could look at that outer `div` as the _host_ of the component.
+
+```js
+// React w/JSX
+render() {
+  return (
+    <div className="my-element">
+      ...
+    </div>
+  )
+}
+```
+
+With custom elements, there is no need to wrap all the HTML inside an outer element to create that host wrapper. The custom element itself is that wrapper and any styles using the `:host` selector will be applied to the whole element. The following example is a HTML template from LitElement using a [tagged template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates). This is NOT JSX.
+
+```js
+// LitElement w/tagged template literal
+render() {
+  return html`
+    <div>
+      ...
+    </div>
+  `;
+}
+```
+
+### Parental awareness
+
+Aside from not needing to wrap all the HTML in yet another `div`, another thing that we get that is a super-power IMHO is parental awareness. What does that mean? Well, a custom element's CSS is fully aware of any changes to the parent custom element. E.g. `<my-element>`.
+
+Where this comes in VERY handy is with applying stylistic changes to your custom element without having to create properties and manage the application and removal of CSS within the scope of the element. The idea is to allow a user to add an attribute to the custom element tag and the element will just respond.
```

Comment:
It would be good to note here that the property needs to be reflected to attributes if you are using them as styling hooks to support frameworks that only set properties (e.g. Svelte & Preact).

Reply:
I know what you are referencing, but I don't remember the fix?

Reply:
I added a comment to the page to come back and address where you made this mention.